tests/product/get-all-products.endpoint.spec.js

```diff
@@ -0,0 +1,25 @@
+import { StatusCodes } from "http-status-codes";
+import request from "supertest";
+import { app } from "../../routes.js";
+
+describe("Get all products", () => {
+  it("should return 200 when the limit and page query params are both positive integers", async () => {
```

Comment:
dont hardcode status codes, use the variable for that:
```suggestion
  it(`should return ${StatusCodes.OK} when the limit and page query params are both positive integers`, async () => {
```